tests/clone_test.go

```diff
@@ -682,24 +682,19 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 							Name: preference.Name,
 							Kind: "VirtualMachinePreference",
 						}
+					})
 
+					DescribeTable("should create new ControllerRevisions for cloned VM", Label("instancetype", "clone"), func(runStrategy virtv1.VirtualMachineRunStrategy) {
+						sourceVM.Spec.RunStrategy = &runStrategy
```

Comment:
If I understand correctly, the current problem is that `Running` and `RunStrategy` cannot co-exist.
We prefer `RunStrategy` and we should stop using `Running` (and even deprecate it).

Therefore, I see you have these two options:
- Fix the problem from the source, i.e. `libvmi`.
- Fix the problem in the most appropiate location for this specific test context: The `BeforeEach` that creates the VM. This means to add there `sourceVM.Spec.Running = nil`.

If fixing the original problem is causing an avalanche of changes all over the codebase, then you can go with the second option for now.

I want to emphasize that it is more important to identify and fix root causes over finishing a specific small cleanup.
It is very common to find bigger problems while working on small scoped changes.

Reply:
Sounds good to me.

Reply:
> I want to emphasize that it is more important to identify and fix root causes over finishing a specific small cleanup.

AFAIK, @iholder101 already [opened an issue](https://github.com/kubevirt/kubevirt/issues/11993) about deprecating the Running field.